pyconfig.h.in

```diff
@@ -1494,6 +1494,9 @@
 /* Define to 1 to build the sqlite module with loadable extensions support. */
 #undef PY_SQLITE_ENABLE_LOAD_EXTENSION
 
+/* "Define if SQLite was compiled with the serialize API" */
```

Comment:
```suggestion
/* Define if SQLite was compiled with the serialize API */
```

Reply:
FYI, this is generated by `AC_DEFINE` in `configure.ac`.